housekeeping/makeBuild.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+
+BUILD_DIR="${BUILD_DIR:?BUILD_DIR variable is not defined}"
+BUILD_TARGET="${BUILD_TARGET:-test}"
```

Comment:
Can we rename it to `FINAL_BUILD_TARGET`. It's very difficult to understand what it is.

Reply:
fixed BUILD_FINAL_TARGET